### PR TITLE
Revert "branching QDM Testcases for qdm specific valdiations"

### PR DESCRIPTION
**Pull request by RohitKandimalla**
This reverts commit f8048a862f0c40c27c1e5749589b812fd939a3f8.

## MADiE PR Template

Jira Ticket: [MAT-0000](https://jira.cms.gov/browse/MAT-0000)
(Optional) Related Tickets:

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included.
* [ ] No extemporaneous files are included (i.e. Complied files or testing results).
* [ ] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/main/java/gov/cms/madie/models/measure/TestCaseGroupPopulation.java [hunk 1]
```diff
@@ -14,7 +14,7 @@
 @Builder(toBuilder = true)
 @NoArgsConstructor
 @AllArgsConstructor
-@ValidScoringPopulation
+//@ValidScoringPopulation
```
Comment (sb-cecilialiu) [inline src/main/java/gov/cms/madie/models/measure/TestCaseGroupPopulation.java:17]: should this be commented out?

Comment (RohitKandimalla) [reply src/main/java/gov/cms/madie/models/measure/TestCaseGroupPopulation.java:17]: Yes, We are removing these validations as a quick way to handle QDM-related changes, There will be a tech debt story to enhance these API validations in the future.

**Review by sb-cecilialiu — APPROVED** [review]
